tests/conformance/scripts/run-tests/runners/stringifyRecords.ts

```diff
@@ -0,0 +1,33 @@
+import { Mcap0Types } from "@foxglove/mcap";
+import stringify from "json-stringify-pretty-compact";
+import { chain, snakeCase } from "lodash";
+
+function replacer(_key: string, value: unknown): unknown {
+  if (value instanceof Uint8Array) {
+    return Array.from(value);
+  }
+
+  if (value instanceof Map) {
+    return Object.fromEntries(value);
+  }
+
+  if (typeof value === "bigint") {
+    return Number(value);
```

Comment:
loss of precision – I guess not a problem today since none of our tests use large uint64 values? but maybe we should have some that do?

Reply:
I think we may want to use strings for uint64 to make JS's life easier. Otherwise I guess we would have to pull in an external json serialization lib that can handle bigints.

Reply:
Yeah I think you're right. I'll change it to a string.